src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -22,6 +22,25 @@
             {% trans "No logic rules for that submission" %}
         
         {% else %}
+            <div class="module filtered" id="changelist">
+                <div class="changelist-form-container">
+                    <div id="changelist-filter">
+                    <h2>{% trans "Filter" %}</h2>
+                        <h3> {% trans "Rule is triggered" %} </h3>
+                        <ul class="logic-rules-filter">
+                            <li>
+                                <a class="logic-rules-filter__link logic-rules-filter__link--all" href="" data-trigger="{% trans 'All' %}" title="{% trans "All" %}">{% trans "All" %}</a>
+                            </li>
+                            <li class="selected">
+                                <a class="logic-rules-filter__link" href="" data-trigger="{% trans 'Yes' %}" title="{% trans "Yes" %}">{% trans "Yes" %}</a>
+                            </li>
+                            <li>
+                                <a class="logic-rules-filter__link" href="" data-trigger="{% trans 'No' %}" title="{% trans "No" %}">{% trans "No" %}</a>
+                            </li>  
```

Comment:
* you typically don't want to include translatable strings into `data-` attributes - they are normally only used for programmatic access.
* I don't think you need the `logic-rules-filter__link--all`, since you can do a `.querySelector('.logic-rules-filter__link[data-trigger="all"]')` in Javascript to filter down on data-attribute as well.
* The `title` attributes don't add any value compared to the link text, so I'd leave them out.
* I'd use `href="#"` so that if the Javascript click capture fails, you don't get a full page reload